kubernetes/opendebates.yaml.j2

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
```

Comment:
You could put the `Secret` (and `ConfigMap`, if we keep it) at the top of this file, to keep everything related to this app in one place.